Orange/widgets/visualize/owmosaic.py

```diff
@@ -321,8 +321,7 @@ def __init__(self):
             gui.comboBox(
                 box, self, value="variable{}".format(i),
                 orientation=Qt.Horizontal, contentsLength=12,
-                callback=self.reset_graph,
-                sendSelectedValue=True, valueType=str, emptyString="(None)")
```

Comment:
`valueType` and `sendSelectedValue` is now detected automatically.

However: this widget must be converted to using models.